Common/Data/Consolidators/DataConsolidator.cs

```diff
@@ -58,7 +58,7 @@ public void Update(IBaseData data)
         /// </summary>
         public IBaseData Consolidated
         {
-            get; private set;
+            get; protected set;
```

Comment:
This change can be avoided, take a look at how `PeriodCountConsolidatorBase` is doing it 👍